all_seaing_navigation/all_seaing_navigation/waypoint_publisher.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+import rclpy
+from rclpy.node import Node
+from geometry_msgs.msg import PoseArray, Pose
+from std_msgs.msg import Header
+from visualization_msgs.msg import Marker, MarkerArray
+import random
+
+class WayPointPublisher(Node):
```

Comment:
Maybe we should call this something like `WaypointPublisherTest` and `waypoint_publisher_test.py`. Feel free to change the name of `waypoint_sender.py` to something like `rviz_waypoint_sender.py` if that's a confusing name.